source/codegen/service_helpers.py

```diff
@@ -107,11 +107,7 @@ def _create_standard_arg(parameter):
         return f"CallbackRouter::handle_callback, "
     elif "callback_token" in parameter:
         return f"handler->token(), "
-    elif (
-        not is_output
```

Comment:
Note: All of these negatives can be captured as the positive `is_size_param_passed_by_ptr`.